Hangman/src/hangman/ColorPickerPanel.java

```diff
@@ -44,14 +45,19 @@ public class ColorPickerPanel extends JPanel implements MouseListener, MouseMoti
 
     JLabel timeStamp;
     JLabel roundNumber;
-
+        
     Ellipse2D.Double q1;
     Ellipse2D.Double q2;
     Ellipse2D.Double q3;
     Ellipse2D.Double q4;
     Ellipse2D.Double q5;
 
     public ColorPickerPanel() {
+        
+        this(new Player());
+        
+        // Could delete stuff under since it does the same thing?
```

Comment:
We might want to consider deleting the extra code because it's implemented in the other constructor